### ISSUE TITLE
required item not working

### ISSUE THREAD

**Issue by Organictuk** [GouveiaXS/angelicxs_free_DrugHarvest#1]
when harvesting it keeps telling me i do not have required item. when i have it in my inventory. does this fully work with qb

**Comment by GouveiaXS** [comment]
Yes, this script fully works with both QB and ESX. The spelling you put in the config most likely does not match the name in your /shared/items.lua there may be an unintended space or other slight error in the config.

**Comment by FacePalmDodger** [comment]
I think its the seplling mistake. 
Replacing all times it says requried with required fixed it for me.
